testutil/coins/margin.go

```diff
@@ -0,0 +1,69 @@
+package coins
+
+import (
+	"testing"
+
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+)
+
+func RequireCoinsDiffInPointOnePercentMargin(t *testing.T, c1, c2 sdk.Coins) {
```

Comment:
Can this be moved to `margin_test.go` as this is a test util func?

Reply:
if moved to a test, it can't be imported by another package